terragrunt/aws/vpc/vpc.tf

```diff
@@ -0,0 +1,38 @@
+module "vpc" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v0.0.46//vpc"
+  name              = "efs-vpc"
```

Comment:
For a name, could you set this to something like `simplify-privacy-statements` just so it's more obvious in the AWS console what it's getting used for.